tools/dump_kv.ts

```diff
@@ -3,13 +3,27 @@
 // Usage: deno run -A --unstable tools/dump_kv.ts
 import { kv } from "@/utils/db.ts";
 
+interface ToReplace {
+  bigint: bigint;
+}
+
+function applyReplacer<T extends keyof ToReplace>(value: ToReplace[T]) {
+  const revivers: { [K in keyof ToReplace]?: (value: ToReplace[K]) => string } =
+    {
+      bigint: (value) => value.toString(),
+    };
+  return revivers[typeof value as T]?.(value) ?? value;
+}
+
```

Comment:
```suggestion
// https://github.com/GoogleChromeLabs/jsbi/issues/30#issuecomment-521460510
function replacer(_key: unknown, value: unknown) {
  return typeof value === "bigint" ? value.toString() : value; // return everything else unchanged
}
```
This has the same effect and is more straightforward 🙂